### PR TITLE
N17: Polyhedron-ops

### PR DESCRIPTION
**Pull request by AngelOnFira**
For #408 :smiley:

### CODE REVIEW

**Review by 17cupsofcoffee — APPROVED** [review]
🚀

**Review by ozkriff — APPROVED** [review]
:+1: (made a few style & fmt tweaks)

**Comment by ozkriff** [comment]
Merged, thanks!

**Comment by ozkriff** [comment]
^ @virtualritz jfyi

**Comment by virtualritz** [comment]
@ozkriff Sorry, just saw this because you tagged me! The entire thread was somehow in my spam. Looking at it now.

File: content/posts/newsletter-017/index.md [hunk 1]
```diff
@@ -606,6 +606,45 @@ _Discussions:
...
+
+Optionally enabled features include:
+
+- Converting a polyhedron [into a bevy Mesh][polyhedron-bevy].
+- Sending a polyhedron to an offline renderer via the [nsi] crate.
+- Writing data out as Wavefront OBJ.
+
+[polyhedron-ops]: https://github.com/virtualritz/polyhedron-ops
+[polyhedron-wiki]: http://en.wikipedia.org/wiki/Conway_polyhedron_notation
+[polyhedron-bevy]: https://github.com/virtualritz/polyhedron-ops/blob/76a0c4b83examples/bevy/bevy.rs
```
Comment (virtualritz) [inline content/posts/newsletter-017/index.md:645]: That link is missing a `/`.
Should be `https://github.com/virtualritz/polyhedron-ops/blob/76a0c4b83/examples/bevy/bevy.rs`, I think

Comment (ozkriff) [reply content/posts/newsletter-017/index.md:645]: oops. will fix in the final draft.

**Pull request merged**